lib/starknet_explorer_web/live/pages/block_detail.ex

```diff
@@ -83,7 +83,8 @@ defmodule StarknetExplorerWeb.BlockDetailLive do
     assigns = [
       block: block,
       view: "overview",
-      verification: "Pending"
+      verification: "Pending",
+      enable_verification: false
```

Comment:
Move this to a config variable we read from an environment variable in `runtime.exs` (with `false` as default value if it's not set), so we can easily switch this feature on deployment